README.md

```diff
@@ -70,13 +70,13 @@ Steps:
     * `platform` - Platform specific files for timer, TLS and threading layers
     * `samples` - The sample applications
     * `src` - The AWS IoT SDK source files
-    * `tests` - Contains tests for verifying that the SDK is functioning as expected. More information can be found [here](https://github.com/aws/aws-iot-device-sdk-embedded-c/blob/master/tests/README.md)
- * View further information on how to use the SDK in the Readme file for samples that can be found [here](https://github.com/aws/aws-iot-device-sdk-embedded-c/blob/master/samples/README.md)
+    * `tests` - Contains tests for verifying that the SDK is functioning as expected. More information can be found [here](https://github.com/aws/aws-iot-device-sdk-embedded-c/blob/master/tests/README.md/)
```

Comment:
This added trailing slashes to the URL. Intended?

Reply:
Yes.  Strangely, when doxygen sees a link ending in .md, it tries to find a .md file on the filesystem to link (which won't be found).  But if it ends in a /, doxygen recognizes it as a URL and links correctly to that.